components/calendar/calendar.js

```diff
@@ -538,8 +538,9 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		});
 	}
 
-	focus() {
+	async focus() {
```

Comment:
This was needed for the focusing on today/the selected date to work when opening the calendar dropdown initially